externs/ima.js

```diff
@@ -16,11 +16,8 @@ var google = {};
 /** @const */
 google.ima = {};
 
-/** @const */
-google.ima.settings = {};
-
-/** @param {string} locale */
-google.ima.settings.setLocale = function(locale) {};
```

Comment:
You deleted this bit because of duplication with line#180/183, is that correct?
Could you please manually verify that calling google.ima.settings.setLocale() still works/compiles correctly without it. If it does, I'm ok with deleting this.

Reply:
Yes I changed the type of the google.ima.settings object so that it 'inherits' all of the ImaSdkSettings methods. That way they only need to be defined once in the ImaSdkSettings class. I was able to compile the code* and call `google.ima.settings.setLocale('fr')` in the local Shaka Player demo app with these changes. Let me know if there's anything else that would be helpful for me to check.

*A note about compiling: I ran into the same issue that the build bot ran into where there was a missing trailing comma. It looks like that problem was introduced with 0845843.